### PR TITLE
Dtsbps 393

### PR DESCRIPTION
**Pull request by gauravchawla73463**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-393

### Change description ###
Add envelope_id to stale-incomplete-blobs report in processor

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/BlobInfo.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+
```
Comment (apereverzin) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/BlobInfo.java:5]: No extra blank line here

Comment (apereverzin) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/BlobInfo.java:5]: No extra blank line here

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    @JsonProperty("file_name")
+    public final String fileName;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonProperty("envelope_id")
+    public final String envelopeId;
+
+    @JsonProperty("created_at")
+    public final String createdAt;
```
Comment (apereverzin) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java:21]: I suggest to make this attribute of type `Instant`

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    @JsonProperty("container")
+    public final String container;
+
+    @JsonProperty("file_name")
+    public final String fileName;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonProperty("envelope_id")
+    public final String envelopeId;
```
Comment (apereverzin) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java:18]: Let's make this attribute of type UUID. This we make code cleaner.

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/BlobInfo.java (hunk 1 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/BlobInfo.java]: done

Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/BlobInfo.java]: removed it

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java (hunk 2 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java]: done

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java (hunk 3 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java]: done

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    @JsonProperty("file_name")
+    public final String fileName;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonProperty("envelope_id")
+    public final UUID envelopeId;
+
+    @JsonProperty("created_at")
+    public final Instant createdAt;
```
Comment (apereverzin) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java:22]: Should also have
`@JsonSerialize(using = InstantSerializer.class)`

Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeInfo.java:25]: done

**Pull request merged**